docs/contributing/documentation-guidelines/index.md

```diff
@@ -35,7 +35,7 @@ There are two ways to preview your modification on a documentation website.
 Follow the steps below.
 
 1. Create a pull request to the repository.
-2. Add the `documentation` label from the sidebar (See below figure).
+2. Add the `deploy-docs` label from the sidebar (See below figure).
 3. Wait for a couple of minutes, and the `github-actions` bot will notify the URL for the pull request's preview.
 
 ![documentation-label](images/documentation-label-for-pull-request.png){ width="800" }
```

Comment:
Would it be possible for you to replace this image as well? :bow:

Reply:
@kenji-miyake I fixed it.